src/neo-vm/ExecutionEngineLimits.cs

```diff
@@ -38,6 +38,11 @@ public sealed record ExecutionEngineLimits
         /// </summary>
         public uint MaxItemSize { get; init; } = 1024 * 1024;
 
+        /// <summary>
+        /// The largest comparable size. If a <see cref="Types.ByteString"/> or <see cref="Types.Struct"/> exceeds this size, comparison operations on it cannot be performed in the VM.
+        /// </summary>
+        public uint MaxComparableSize { get; init; } = 65536;
```

Comment:
```suggestion
        public uint MaxComparableSize { get; init; } = ushort.MaxValue;
```

Reply:
@shargon I think it's better to revert this. Because all other fields are powers of base 2.